account/accounts.go

```diff
@@ -33,6 +33,14 @@ var (
 	ErrAccountKeyStoreMissing         = errors.New("account key store is not set")
 )
 
+type ErrCannotLocateKeyFile struct {
+	Msg string
+}
+
+func (e ErrCannotLocateKeyFile) Error() string {
+	return e.Msg
+}
+
```

Comment:
Implementing the error interface is a legitimate path to achieving what you are doing here. But I thought I'd also introduce you to the concept of `Error Equality` via `Error Wrapping`. See https://trstringer.com/errors-and-error-wrapping-go/#error-equality for a nice breakdown of how it works.

I'm not asking for any changes at all here, but I know you'd appreciate this feature of Go